src/components/PageIndicator.tsx

```diff
@@ -50,19 +50,20 @@ const PageIndicator: React.FC<PageIndicatorProps> = ({
 
     const getMessage = (moduleIndex: number) => {
         if (module === moduleIndex) {
-            return "Are you sure to restart this module? You will lose all progress.";
+            return "Are you sure you want to restart this module? You will lose all progress.";
         } else {
             return "Are you sure you want to leave this module? You will lose all progress.";
         }
     };
 
     return (
         <Flex align="center" justify="flex-end" className={styles.container}>
-            {map(moduleNames, (name, index) => {
+            {map(moduleNames, (name, index: string) => {
                 const moduleIndex = Number(index);
```

Comment:
Will there ever be a case where `index` is a non-numeric string?

Reply:
No, the Module type is an enum